concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/00 Getting Started with SelectBox.md

```diff
@@ -1,3 +1,5 @@
+#include uiwidgets-editors-choose-a-dropdown-editor
+
 #include tutorials-intro-installationnote
 
```

Comment:
Line 9 (below):
`Each section in this tutorial covers a single configuration step.`

Can we change "covers" to "describes"?

Reply:
Noted